apps/aecore/src/aec_parent_connector.erl

```diff
@@ -415,30 +420,30 @@ get_pinning_data() ->
     end.
 
 get_generations(FromHeight, ToHeight) ->
```

Comment:
I think this function should be called `get_microblocks_between_heights` - a generation contains both keyblocks and microblocks 🤔